pages/registration.tsx

```diff
@@ -0,0 +1,24 @@
+import Head from "next/head";
+import { LayoutRoot } from "~/components/layout/home";
+import { ContentBoard } from "~/components/ui/board/content-board/content-board";
+import { FormTitle } from "~/components/ui/form/form-title/form-title";
+import { FormSubtitle } from "~/components/ui/form/form-subtitle/form-subtitle";
```

Comment:
```suggestion
import { ContentBoard } from "~/components/ui/board/content-board";
import { FormTitle } from "~/components/ui/form/form-title";
import { FormSubtitle } from "~/components/ui/form/form-subtitle";
```

Can we compile these exports into each of its root folder? Having the import directory doubled `/content-board/content-board` doesn't feel like good UX.

Reply:
Did you mean instead of having `~/components/ui/board/content-board/content-board.tsx`, we have `~/components/ui/board/content-board/index.tsx`?

Reply:
@adityapurwa I'm thinking we create an `index.ts` file and pass all the exports through there, like this:

```
export * from "./content-board";
```

Reply:
I personally avoid having two ways of importing something.

By grouping and exports into an `index.ts`, we'd now have two ways of importing. Directly to the component file, and using the index groupings. This would introduce inconsistencies.

It will also be problematic with the go-to source feature, as now clicking on the reference will take us to the index file instead of directly to the file of interest.

What do you think about it? Let me know if the decision stands to use export groupings. 🙌

Reply:
@adityapurwa If you think the `content-board` folder only contains this one file, I think it's okay to move the `content-board.tsx` file one folder up (`~/components/ui/board/content-board.tsx`).

The export grouping I think is only necessary when the folder contains a lot of components. Since export grouping is already used extensively at WBW, and we want to keep our styleguide as close to it as possible, we'll eventually have to use export grouping when we want to group exports of design system components (e.g. those at `~/components/ui/*`).

Reply:
Yes you are correct, I agree we can lift the component file outside of its folder. Even when we adds *.test later, I don't think it'd be too crowded. Let's go with uplifting it then 🚀

Reply:
> Even when we adds *.test later, I don't think it'd be too crowded.

We store tests in a separate `__tests__` folder so it wouldn't be a problem regardless.